connection/connection.go

```diff
@@ -203,11 +210,33 @@ func (cmm ExtendedCSIMetricsManager) RecordMetricsClientInterceptor(
 	start := time.Now()
 	err := invoker(ctx, method, req, reply, cc, opts...)
 	duration := time.Since(start)
-	cmm.RecordMetrics(
-		method,   /* operationName */
-		err,      /* operationErr */
-		duration, /* operationDuration */
-	)
+
+	if cmm.HaveAdditionalLabel(metrics.LabelMigrated) {
+		// record migration status
+		additionalInfo := ctx.Value(AdditionalInfoKey)
+		migrated := "false"
+		if additionalInfo != nil {
+			migrated = additionalInfo.(AdditionalInfo).Migrated
+		}
+		cmmv, metricsErr := cmm.WithLabelValues(map[string]string{metrics.LabelMigrated: migrated})
+		if metricsErr != nil {
+			klog.Errorf("Failed to record migrated status, error: %v", metricsErr)
+		} else {
+			cmmv.RecordMetrics(
```

Comment:
Can we save a few lines and have both paths share this call?

Reply:
I would love to but the caller is different... The cmmv is a different thing with cmm. Do you have any suggestion on how can I combine these two?

Reply:
Instead of creating a new cmmv, can you set the return value to cmm?

Reply:
Done